test/fixtures/adapters/swagger20/api-description-example.js

```diff
@@ -0,0 +1,1228 @@
+export default {
```

Comment:
Fixed. It was easier to test loading from JS, but makes sense to be actual JSON that we parse.